tutorials/simple/arb_with_all/out

```diff
@@ -0,0 +1,178 @@
+Compiling simple:composed:simple_arb_all
```

Comment:
uhhhhhhhhhhhhhhhhhhhhh should this have been committed? notably contains `/home/lucas`...

Reply:
Oops.